_topics/en/multisignature.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Multisignature
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - 2pECDSA
+  - Two-Party ECDSA (2pECDSA)
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Multisignatures** are digital signatures that look like they were
+  created using only a single private key but which were actually
```

Comment:
I'm not sure we should use the framing that the default is using a single private key and that multisignature "looks like" that default. In the Bitcoin we're trying to build, most outputs/spends are single pubkey/signatures, and they're created using lots of different techniques which are opaque to the blockchain. In that world, it doesn't make sense to say that multisignatures look like single signer signatures any more than saying single signer signatures look like multisignatures.